src/dmd/declaration.h

```diff
@@ -442,14 +442,45 @@ enum ILS
 
 enum BUILTIN
 {
-    BUILTINunknown = -1,        // not known if this is a builtin
-    BUILTINno,                  // this is not a builtin
-    BUILTINyes                  // this is a builtin
+    BUILTINunknown = -1,    /// not known if this is a builtin
```

Comment:
Prefer to not use mixed sign for enum, as that interferes with efficient switch dispatch and some other optimizations.

Reply:
That was there before and I'd rather not make unnecessary changes. Feel free to follow up on that though.